realm/src/main/java/io/realm/Realm.java

```diff
@@ -895,6 +904,80 @@ public void writeCopyTo(File destination) throws IOException {
         return result;
     }
 
+    /**
+     * Converts a RealmObject that has been created using it's standard constructors instead of
+     * {@link #createObject(Class)} to a RealmObject that is managed by the Realm.
+     *
+     * To persist any further changes, the returned object must be be used.
+     *
+     * @param object RealmObject to copy to the Realm.
+     * @return The converted RealmObject that has all its properties managed by the Realm.
```

Comment:
Please consider  rephrase: A managed RealmObject with properties backed by the Realm.